config/locales/devise.fr.yml

```
@@ -29,8 +29,9 @@ fr:
       hello: Bonjour %{user_full_name},
       invitation_instructions:
         accept: Choisir mon mot de passe
+        accept_user_invitation_instruction: 'Pour rejoindre la communauté, choisissez votre mot de passe :'
```

```suggestion
        accept_user_invitation_instruction: 'Pour rejoindre la communauté, choisissez votre mot de passe :'
```
J'ai remplacé par un espace insécable